webindex/modules/data/src/test/java/webindex/data/fluo/PageLoaderTest.java

```diff
@@ -0,0 +1,63 @@
+package webindex.data.fluo;
+
+import java.net.MalformedURLException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import webindex.core.models.Page;
+import webindex.core.models.URL;
+
+@SuppressWarnings("unused")
```

Comment:
There's no reason to suppress this warning, since this is easily avoidable. For example, replace:
```java
Page p = null;
PageLoader loader = PageLoader.updatePage(p);
```

with:

```java
PageLoader loader = PageLoader.updatePage((Page) null);
```

Reply:
Thank you, fixed.